### ISSUE TITLE
changing country firing input "change" event, while its not

### ISSUE THREAD

**Issue by iamstarkov** [jackocnr/intl-tel-input#344]
```js
// always fire the change event as even if nationalMode=true (and we haven't updated the input val),
// the system as a whole has still changed - see country-sync example. think of it as making
// a selection from a select element.
this.telInput.trigger("change");
```
[code][code] and [demo][demo].

I developed UI and heavily relied on `change`, `blur` and `focus` to verify it, so this event breaks contract of native events and spoils everything. I checked out country sync demo and fully understand purpose of adding this, so can we go further and emit not native `change` event, but maybe custom `country-change` event, on which everybody still can rely. Solution will a bit more pure about events. What do you think?

[code]: https://github.com/jackocnr/intl-tel-input/blob/master/src/js/intlTelInput.js.ejs#L1091-L1092
[demo]: http://jackocnr.com/lib/intl-tel-input/examples/gen/country-sync.html

**Comment by jackocnr** [comment]
Yer I agree it's not consistent with the native `change` event, so we should rename it. I don't have time to make this change, but I'm open to pull requests. Cheers.

**Comment by jackocnr** [comment]
Fixed in v7.1.0.
